locales-pending/plus-shutdown.ftl

```diff
@@ -0,0 +1,14 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+plus-shutdown-banner-button-learn-more = Learn more
+plus-shutdown-banner-button-close = Close
+plus-shutdown-banner-free-runup-heading = Update: { -brand-monitor } data broker scan results will be deleted in 30 days
+plus-shutdown-banner-free-runup-subheading = View your results ahead of the the { -brand-monitor-plus } shutdown.
+plus-shutdown-banner-free-shutdown-heading = Data broker scan results have been deleted, but { -brand-monitor } lives on
+plus-shutdown-banner-free-shutdown-subheading = You can still continue to access data breach monitoring to help keep your info safe online.
+plus-shutdown-banner-plus-runup-heading = Data broker scan results and in-progress removals will be deleted
+plus-shutdown-banner-plus-runup-subheading = View your dashboard before we say goodbye to { -brand-monitor-plus } for good on December 17, 2025.
```

Comment:
Could we add a var here instead?

Reply:
I debated that, but then I was worried about messing up the formatter, or doing something wrong with the date object, so I opted to hope for it either not changing, or us remembering to update this.